documents/templates/documents/document_edit.html

```diff
@@ -18,13 +19,25 @@
                 Essaye de rendre le titre clair pour que d'autres puissent trouver ton document.
             </div>
         </div>
+
         <div class="mb-3">
             <label for="id_tags" class="form-label">Tags</label>
             {{ form.tags }}
             <div class="form-text">
                 Sélectionne si possible les tags qui correspondent à ton document.
             </div>
         </div>
+
+        {% if request.user|has_moderation_perm_on:doc %}
+            <div class="mb-3">
+                <label for="id_tags" class="form-label">Certifié: </label>
```

Comment:
This lacks explanation i think. We should explain to the user, what is "certifié" (maybe even find a better name) and explain also that it's only because the user is a moderator that they can certify it

Reply:
I don't think it's an issue since it's the only thing a moderator is able to do now, if we give them the role it's only for that so they will know what is it for.